packages/apollo/lib/errors/authentication.error.ts

```diff
@@ -1,6 +1,8 @@
 import { GraphQLError, GraphQLErrorOptions } from 'graphql';
 
 /**
+ * @publicApi
```

Comment:
publicApi directive should be located at the end of the description comment

Reply:
Done! see commit here: [b1ec547](https://github.com/nestjs/graphql/pull/3475/commits/b1ec5472a05388c874fe09d764a455580dcc275f), I will also update the other PR these days 😉